### ISSUE TITLE
Chapter: mathematical foundation of Word2Vec Model

### ISSUE THREAD

**Issue by antoniogamiz** [antoniogamiz/tfg#24]
This includes:

- Gradient descent
- Backpropagation
- Probalistic concepts? I'm still not sure about this one. Maybe I will move it to another place.

**Comment by antoniogamiz** [comment]
Duplicada de #38
